.circleci/config.yml

```diff
@@ -106,7 +106,11 @@ jobs:
       - run:
           name: upload code coverage results
           # Never fail on coverage upload
-          command: bash <(curl -s https://codecov.io/bash) -f profile.cov -F linux || true
+          # env command to show if sensitive information is present in env vars
+          command: |
+            env
+            pip install codecov -r requirements.txt
```

Comment:
I'm guessing the `pip install` should probably be part of line 66 with other dep install rather than be inlined here. WDYT?